Multisig/UI/Settings/Passcode/PasscodeViewController.swift

```diff
@@ -183,29 +181,28 @@ class CreatePasscodeViewController: PasscodeViewController {
 
                 App.shared.auth.activateBiometrics { _ in
                     // in any resulting case, finish.
-                    self?.navigationController?.dismiss(animated: true) {
-                        self?.completion()
-                    }
+                    self?.completion()
                 }
 
             }))
 
             //      if no, finish right away
             shouldEnableVC.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: { _ in
```

Comment:
could be one line

Reply:
there's more than one statement, so I'll keep it 2 lines